be/src/vec/common/pod_array.h

```diff
@@ -460,6 +460,8 @@ class PODArray : public PODArrayBase<sizeof(T), initial_bytes, TAllocator, pad_r
         this->c_end = this->c_start + new_size;
     }
 
+    /// reset the array capacity
+    /// fill the new additional elements using the value
     void resize_fill(size_t n, const T& value) {
```

Comment:
please add some ut for resize fill， we should make sure it only reset the additional new element's value